active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+
+
+Additionally, the preset Severity colors have already been designed for theme changes, 
+high-contrast mode, color confusion accessibility, and contrast with foreground colors. 
+We recommend to use these colors when possible and to include custom logic in your application to
+adapt to the various color states and accessibility features.
+
+
+Please view the UX guidance for
+[Standard Icons](https://docs.microsoft.com/en-us/windows/win32/uxguide/vis-std-icons) 
+and [Color](https://docs.microsoft.com/en-us/windows/win32/uxguide/vis-color) 
+to ensure your message is communicated clearly and accessible to users.
+
+
+### Severity
+ Avoid setting the Severity property for a notification that does not match the information communicated in 
+ the Title, Message, or custom content.
+ 
+
+ The accompanying information should aim to communicate the following to use that Severity.
+ - Error: An error or problem that has occurred.
+ - Warning: A condition that might cause a problem in the future.
+ - Success: A long-running and/or background task has completed.
+ - Default: General information that requires the user's attention.
+
+Icons and color should not be the only UI components signifying meaning for your notification. 
+Text in the notification's Title and/or Message should be included to display information.
+
+
+## Message 
+
+Text in your notification will not be a constant length in all languages. For the Title and Message property 
+this may impact whether your notification will expand to a second line. 
+We recommend you avoid positioning based on message length or other UI elements set to a specific language.
+
+The notification will follow standard mirroring behavior when localized to/from languages that are right to left (RTL) or left to right (LTR). The icon will only mirror if there is directionality.
+
+Please view the guidance for 
+[Adjust layout and fonts, and support RTL](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl) 
+for more information about text localization in your notification.
+
+
+# API Notes
+
+### Notable Properties  
+| Name | Description |
+|:-:|:--|
+| IsOpen| Gets or sets a value that determines the visibility of the InfoBar. By default, is set to false. |
+| Severity | Gets or sets a value that indicates the  color and icon to style the InfoBar. It will also define the assistive technology settings. |
+| IsUserDismissable| Gets or sets a boolean that indicates whether the user will be able to dismiss the InfoBar.
+
+
+### Events  
+| Name | Description |
+|:-:|:--|
+| CloseButtonClick | Occurs after the close button has been clicked. |
+| Closed | Occurs after the info bar is closed. |
+| Closing |Occurs just before the info bar begins to close. |
+
+
+# API Details
+
+```c++
+enum InfoBarCloseReason
+{
+    CloseButton = 0,
+    Programmatic = 1,
+};
+
+enum InfoBarSeverity
+{
+    Error = 0,
```

Comment:
Usually the "default" value is set as the `0` value, and so `Default` should be `0`.

Reply:
Is Default the default?